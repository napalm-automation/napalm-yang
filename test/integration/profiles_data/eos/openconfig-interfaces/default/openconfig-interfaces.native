{"interfaces": {"Management1": {"lastStatusChangeTimestamp": 1497732472.6491787, "name": "Management1", "interfaceStatus": "connected", "autoNegotiate": "success", "description": "", "burnedInAddress": "08:00:27:6b:fa:fe", "bandwidth": 1000000000, "mtu": 1500, "hardware": "ethernet", "duplex": "duplexFull", "loopbackMode": "loopbackNone", "physicalAddress": "08:00:27:6b:fa:fe", "interfaceCounters": {"outBroadcastPkts": 0, "linkStatusChanges": 5, "totalOutErrors": 0, "inMulticastPkts": 0, "counterRefreshTime": 1497734297.210793, "inBroadcastPkts": 19, "inputErrorsDetail": {"runtFrames": 0, "fcsErrors": 0, "alignmentErrors": 0, "rxPause": 0, "symbolErrors": 0, "giantFrames": 0}, "inOctets": 97807, "outDiscards": 0, "outOctets": 112236, "inUcastPkts": 0, "outputErrorsDetail": {"lateCollisions": 0, "deferredTransmissions": 0, "txPause": 0, "collisions": 0}, "outUcastPkts": 758, "outMulticastPkts": 0, "totalInErrors": 0, "inDiscards": 0}, "interfaceStatistics": {"inBitsRate": 327.13344777831077, "updateInterval": 300.0, "outBitsRate": 513.747345263329, "outPktsRate": 0.3882478230578469, "inPktsRate": 0.00013831271000537475}, "interfaceAddress": [{"secondaryIpsOrderedList": [], "primaryIp": {"maskLen": 24, "address": "10.0.2.15"}, "broadcastAddress": "255.255.255.255", "secondaryIps": {}, "virtualIp": {"maskLen": 0, "address": "0.0.0.0"}}], "lineProtocolStatus": "up", "forwardingModel": "routed"}, "Ethernet1": {"lastStatusChangeTimestamp": 1497732461.1365635, "name": "Ethernet1", "interfaceStatus": "connected", "autoNegotiate": "unknown", "description": "", "bandwidth": 0, "mtu": 1500, "hardware": "ethernet", "duplex": "duplexFull", "loopbackMode": "loopbackNone", "physicalAddress": "08:00:27:66:ef:40", "interfaceCounters": {"outBroadcastPkts": 0, "linkStatusChanges": 1, "lastClear": 1497732401.4158947, "inMulticastPkts": 97, "counterRefreshTime": 1497734297.214895, "inBroadcastPkts": 2630, "inputErrorsDetail": {"runtFrames": 0, "fcsErrors": 0, "alignmentErrors": 0, "rxPause": 0, "symbolErrors": 0, "giantFrames": 0}, "inOctets": 212355, "outDiscards": 0, "outOctets": 43559, "inUcastPkts": 118, "outputErrorsDetail": {"lateCollisions": 0, "deferredTransmissions": 0, "txPause": 0, "collisions": 0}, "outUcastPkts": 104, "outMulticastPkts": 73, "totalInErrors": 0, "inDiscards": 0, "totalOutErrors": 0}, "interfaceStatistics": {"inBitsRate": 0.0, "updateInterval": 300.0, "outBitsRate": 0.0, "outPktsRate": 0.0, "inPktsRate": 0.0}, "interfaceAddress": [{"secondaryIpsOrderedList": [], "primaryIp": {"maskLen": 24, "address": "192.168.50.11"}, "broadcastAddress": "255.255.255.255", "secondaryIps": {}, "virtualIp": {"maskLen": 0, "address": "0.0.0.0"}}], "lineProtocolStatus": "up", "forwardingModel": "routed"}}}
