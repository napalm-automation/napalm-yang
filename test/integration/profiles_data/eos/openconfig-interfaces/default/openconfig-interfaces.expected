{
    "interfaces": {
        "interface": {
            "Ethernet1": {
                "name": "Ethernet1",
                "state": {
                    "admin-status": "UP",
                    "counters": {
                        "in-broadcast-pkts": 2630,
                        "in-discards": 0,
                        "in-errors": 0,
                        "out-unicast-pkts": 104
                    },
                    "description": "",
                    "enabled": true,
                    "mtu": 1500,
                    "oper-status": "UP",
                    "type": "ethernetCsmacd"
                }
            },
            "Management1": {
                "name": "Management1",
                "state": {
                    "admin-status": "UP",
                    "counters": {
                        "in-broadcast-pkts": 19,
                        "in-discards": 0,
                        "in-errors": 0,
                        "out-unicast-pkts": 758
                    },
                    "description": "",
                    "enabled": true,
                    "mtu": 1500,
                    "oper-status": "UP",
                    "type": "ethernetCsmacd"
                }
            }
        }
    }
}
